src/wallet/wallet.h

```diff
@@ -1308,6 +1313,10 @@ class CWallet final : public CCryptoKeyStore, private interfaces::Chain::Notific
 
     /** Implement lookup of key origin information through wallet key metadata. */
     bool GetKeyOrigin(const CKeyID& keyid, KeyOriginInfo& info) const override;
+
+    int GetLastBlockHeight() const { return m_last_block_processed_height; };
```

Comment:
In commit "Add m_last_block_processed_height field in CWallet" (0284c72ab07362150fd26c86966ea7c7bc6cfed5)

Can this `assert(m_last_block_processed_height >= 0)`? This would catch usage errors if there are calls to this function before the wallet is initialized.

Reply:
Check should be there in b1ae297

Reply:
Note : I've seen one Travis failure on feature_reindex.py due to the assert. I wasn't able to reproduce it on my local dev env, I think it's caused by Travis really slow setup, but that's not great, what's your take on it ?